angular-apollo-tailwind/src/main.ts

```diff
@@ -11,4 +11,3 @@ if (environment.production) {
 platformBrowserDynamic()
   .bootstrapModule(AppModule)
   .catch((err) => console.error(err));
```

Comment:
Nitpick: Removing unrelated and likely unintended whitespace change.

```suggestion
platformBrowserDynamic()
  .bootstrapModule(AppModule)
  .catch((err) => console.error(err));

```